webrender/src/renderer.rs

```diff
@@ -699,6 +699,11 @@ impl CacheRow {
 /// The bus over which CPU and GPU versions of the GPU cache
 /// get synchronized.
 enum GpuCacheBus {
+    /// Persistently mapped buffer-based updates
+    #[cfg(not(feature = "gleam"))]
+    PMbuffer {
+        mapped_ranges: Vec<std::ops::Range<u64>>,
```

Comment:
all the mapped ranges tracking could go away if we require the coherency, which is always available for buffers

Reply:
Removed the ranges in https://github.com/szeged/webrender/pull/297/commits/76b255b4b7ec31e659242e04dbd757177777abf4#diff-f5062b694b9fe53fc1757ed483d577d9R704